playbooks/rfcs.md

```diff
@@ -65,14 +83,25 @@ Create an issue, and work from this template:
 
     You can see our discussion [in slack here](/link/to/slack.com)
 
-Give a week.
+    ## How this RFC is implemented
+
+    As a trial, we'll add a new Peril rule that implements spellchecking on every PR that includes
+    a Markdown document in README. We'll then gather asynchronous feedback from engineers about the
+    new Peril rule and how it impacted their workflow. We'll confirm we want this rule and incorporate
+    any further changes to the original RFC. We'll then roll out the Peril rule to all systems.
+
+    ## How this RFC is resolved
+
+    This RFC will be considered approved when 50% of the engineers at Artsy actively approve this proposal.
+
+Give a week for discussion.
 
 ### Resolution
 
-Resolve the RFC, you can work with this template:
+To resolve the RFC you can work with this template:
 
     ## Resolution
-    We decided to do it.
+    We decided to do it. 50% of the engineering team actively approved of this change.
 
     ## Level of Support
     3: Majority acceptance, with conflicting feedback.
```

Comment:
(Commenting here only because I can't comment on the relevant lines below.)

This PR supersedes the recommendation below, so that can be removed:

> Resolving an RFC requires you to have some nuance about the feedback. If it seems to be unresolved, or still active a week later then calling a town hall style meeting for people involved will probably shake out some come kind of resolution.